scripts/test.py

```diff
@@ -7,39 +7,44 @@
 This script will also generate a JUnit XML file, which can be used to integrate
 with CI/CD pipelines.
 
-Usage: test.py [-h] [-m] [-v] [--version] [dir]
+Usage: test.py [-h] [-m] [-s] [--version] [--no_clean | --coverage] [dir]
 
 Example usage: scripts/test.py compiler_tests/_example
 
 This will print out a progress bar and only run the example tests.
 The output would be placed into bin/output/_example/example/.
 
-For more information, run scripts/test.py -h
+For more information, run scripts/test.py --help
 """
 
 
 __version__ = "0.1.0"
 __author__ = "William Huynh (@saturn691)"
 
 
-import argparse
 import os
+import argparse
 import shutil
 import subprocess
 import queue
 from pathlib import Path
 from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import List, Optional
+from colorama import init, Fore
+init(autoreset=True)
```

Comment:
`init()` is actually here to trigger the `autoreset` option (i.e., not needing to reset style to default after each style changing call, which makes for cleaner code imo).
However, I added `just_fix_windows_console()` "just" in case heh